utils/data.ts

```diff
@@ -132,7 +132,7 @@ export const transactionTabs: TransactionData = {
       lazlo: 'pending',
     },
     input: {
-      sats: '101,000',
+      sats: '100,000',
```

Comment:
I haven't played through the last part of chapter 10 yet ("Making a payment"). I'm not 100% all these numbers are correct. Going to play through it now and circle back to review this last set of numbers.

Reply:
should we now say that you end up with 98,000 when paying the 1,000 sat to laszlo?

Reply:
Here's a comment from MZ on this: https://github.com/saving-satoshi/saving-satoshi/pull/1174#discussion_r1842343476
![Screenshot from 2024-11-14 13-37-26](https://github.com/user-attachments/assets/28d5cd6f-c079-4f4e-aac1-2e74e5b5b7a5)

Here's my understanding:
- You start with 101,000 sats
- The 2-of-2 multisig has `input`: 101,000 sats, `output`: 100,000 sats because 1,000 sats went to miner fees
- The commitment transaction should always have `input`: 100,000 sats, and the `output` should add up to 99,000 sats because we reserve 1,000 sats for fees
- When you buy a beer from Laszlo, the commitment transaction works out to be `input`: 100,000, `output`: 98,000 sats to you, and 1,000 sats to Laszlo. You most recent comment seems correct to me.

Reply:
I reviewed the numbers you updated in 788751e65ddc028de04bf8d084fbd12a9c3f5890 and they look good! Resolving this comment.